src/controller/python/chip-device-ctrl.py

```diff
@@ -635,6 +636,17 @@ def echo_alive(message):
     print(message)
     return message
 
+def resolve(fabric_id: int, node_id: int) -> Dict[str, Any]:
```

Comment:
This is part of Nirav's change, it will disappear once Nirav's change is merged into test_event2.2 and I do a merge onto that.